pkg/controller/installation/installation_controller.go

```diff
@@ -220,7 +212,7 @@ func (r *ReconcileInstallation) processStage(instance *v1alpha1.Installation, pr
 		if !(phase == string(v1alpha1.PhaseCompleted)) {
 			incompleteStage = true
 		}
-		reconciler, err := products.NewReconciler(r.context, v1alpha1.ProductName(product), r.client, r.restConfig, r.coreClient, configManager, instance)
```

Comment:
the context has to be the first argument.

Reply:
It is passed to `reconcile`, it doesn't seem to be needed here

Reply:
oh, I thought it was moved, didn't realise it was just removed!